### ISSUE TITLE
git 'stable' branch/tag points to old v1.42 release

### ISSUE THREAD

**Issue by eharris** [rclone/rclone#5811]
I'm pretty sure the latest stable isn't v1.42.

**Comment by eharris** [comment]
The install docs mention the `:latest` tag for the latest stable version, but that doesn't seem to work:

```
$ git clone -b latest https://github.com/rclone/rclone
Cloning into 'rclone'...
fatal: Remote branch latest not found in upstream origin
```

**Comment by eharris** [comment]
Ok, then what's the preferred tag/branch name to use to get the latest stable code?

**Comment by ncw** [comment]
You need to use the latest release tag so `v1.57.0` we don't maintain a `latest` branch or tag at the moment.

This is exactly the thing that `go install github.com/rclone/rclone@latest` is supposed to do for you but that doesn't work because of reasons in the other thread.

**Comment by eharris** [comment]
Yes, I'm trying to build latest stable (rather than git master branch) and `go get` isn't working (see #5810)

**Comment by ivandeex** [comment]
This oneliner will find you the tag :)
```
$ basename $(curl -sLo/dev/null -w"%{url_effective}" https://github.com/rclone/rclone/releases/latest)
v1.57.0
```

**Comment by ivandeex** [comment]
As for the `stable` branch.
The project has a number of archived branches. Don't use them.
If you find traces in the docs, we'll clean it up.

**Comment by ivandeex** [comment]
And here is the full script
```
latest_release_url=$(curl -sLo/dev/null -w"%{url_effective}" https://github.com/rclone/rclone/releases/latest)
latest_tag=$(basename $latest_release_url)
git clone --depth 1 -b $latest_tag https://github.com/rclone/rclone
cd rclone
go build
cp ./rclone /your/target/location
```

**Comment by ivandeex** [comment]
Ultimately, we can free developers from shell scripting and add a step like below in the `release` CI job:
```
git tag -f latest ${CURRENT_RELEASE_TAG}
git push --force latest
```

@ncw ??

**Comment by eharris** [comment]
I would appreciate if a `latest` or similar tag were included in the build/release process.
As for the `stable` tag, it should probably be removed (or updated) to avoid potential confusion.

**Comment by ncw** [comment]
I've removed the `stable` branch from the repo now - apologies for the confusion.

Whether we have a latest/stable tag I'm not so sure.

There is more than one thing we might want to tag as stable...

- current release v1.57.0
- current point release v1.57.1
- current stable branch v1.57-stable which is v1.57.1 plus possibly unreleased patches destined for v1.57.2

Ultimately the way to see what the releases were built on is to do

```
$ curl -s https://api.github.com/repos/rclone/rclone/releases/latest | grep tag_name
  "tag_name": "v1.57.0",
```

or

```
$ curl -s https://downloads.rclone.org/version.txt
rclone v1.57.0
```

**Comment by eharris** [comment]
Thanks.  To my view, the latest/stable tag should point to whatever the normal "stable" released version is, which to me would mean the current stable point release, but not including unreleased patches.

**Comment by animosity22** [comment]
Is this done or more to do here?

**Comment by ivandeex** [comment]
@animosity22 
We are waiting for @ncw to decide

either `doc fix` in CONTRIBUTING.md: https://github.com/rclone/rclone/issues/5811#issuecomment-967338220 and/or https://github.com/rclone/rclone/issues/5811#issuecomment-971803192

or `build` enhancement: https://github.com/rclone/rclone/issues/5811#issuecomment-967353168
